File: swagger/models/upload_client_document_response.py
from .base_model import BaseModel


class UploadClientDocumentResponse(BaseModel):
	"""
	no description available

	Attributes:

		file_size
		The size of the uploaded file.

		file_name
		The name of the uploaded file.
	"""

	input_map = {
		'FileSize': 'file_size',
		'FileName': 'file_name',
		}

	output_map = {
		'file_size': 'FileSize',
		'file_name': 'FileName',
		}

	file_size = None
	file_name = None

File: swagger/Models/UploadClientDocumentResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class UploadClientDocumentResponse extends BaseModel
{
	/**
	 * The size of the uploaded file.
	 * 
	 * @var integer format:int64
	 */
	public $fileSize;
	/**
	 * The name of the uploaded file.
	 * 
	 * @var string 
	 */
	public $fileName;

	protected function getInputMap()
	{
		return [
			'FileSize' => ['fileSize', 'integer', 'int64'],
			'FileName' => ['fileName', 'string', null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'fileSize' => ['FileSize', 'integer', 'int64'],
			'fileName' => ['FileName', 'string', null],
			];
	}
}
